coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@praveenyr please add /coverage/* to your .gitignore file and delete the coverage folder.